include/boost/geometry/algorithms/detail/overlay/get_turns.hpp

```diff
@@ -327,10 +327,19 @@ private :
     typedef typename model::referring_segment<point1_type const> segment1_type;
     typedef typename model::referring_segment<point2_type const> segment2_type;
 
-    template <typename Iterator, typename RangeIterator, typename Section, typename RobustPolicy>
+    template
+    <
+        typename Iterator, typename RangeIterator,
+        typename Section,
+        typename IntersectionStrategy, typename RobustPolicy
+    >
     static inline void advance_to_non_duplicate_next(Iterator& next,
```

Comment:
I moved this. So this will give a merge conflict later.

Reply:
No problem, I'll resolve them.